### PR TITLE
Huge handoff/matchups update

### LINKED ISSUE #9: Handoff summary using Guatemalan, El Salvadoran, Honduran lakes
In order to have a better dataset for handoffs between 5-7, 7-8 and 7-9, we need more lakes. Instead of applying data from the US, we'll create our own regional handoff.

### PR DESCRIPTION
**Pull request by steeleb**
Okay Matt!

Here's the PR for the scripts/etc I mentioned in the email on Friday. It's huge, but most of it is redundant of previous PRs/PRs from ROSS_RS_mini_tools.

### File overview:
- `GEE_scripts/guat_elsal_hon_getCenters.js`: this is a modified version of Xiao's Chebyshev centers script -- I modified some of the parameters for use in our Rmd scripts, but this is nearly verbatim Xiao's script.
- `landsat_c2/forHandoffs/LandsatC2_M4-7_SurfaceRefTempStacks.Rmd`, `landsat_c2/forHandoffs/LandsatC2_M8-9_SurfaceRefTempStacks.Rmd`, `landsat_c2/forHandoffs/pySetup.R` are dupes of the `landsat_cs/literate_code/` folders, but they use the Chebyshev centers produced by the above script.
- `landsat_c2/literateCode/LandsatC2_M8-9_SurfaceRefTempStacks.Rmd` - small tweaks to clean up the script.
- `programs/1_RS_collate_harmonize.Rmd` small tweaks to account for additional GEE output in same folder (aka, more files from the `landsat_c2/forHandoffs` scripts
- `programs/3_regionalRS_collate_harmonize.Rmd` - this is the same as the script `programs/1_RS_collate_harmonize.Rmd`, except it does collation of the regional Chebyshev center data.
- `programs/4_regional_handoff_calcs.Rmd` - tweaks to account for more data from the regional Landsat stack.
- `programs/5_apply_handoff_coefficients.Rmd` - little file to apply coefficients from `programs/4_regional_handoff_calcs.Rmd` to the Yojoa LS stack
- `programs/6_add_era5_data.Rmd` - little file to add the ERA5 data to the Yojoa stack
- `programs/7_LSC2_secchi_matchup.Rmd` - edited this to overmatch and filter by date window
- I updated a bunch of the README's to clean up the repo

### Progress:
Closes #9

### CODE REVIEW

**Comment by steeleb** [comment]
@kathryn-willi will you please look at the README files in these to make sure they make sense?? This is an example workflow we're working on to show the power of RS data with limited in situ data. THANK YOU!

File: README.md [hunk 1]
```diff
@@ -2,34 +2,52 @@
...
 
-`landsat_c2` contains the scripts used to run the GEE commands and pull the stacks. The raw output of these scripts have been saved in the `data/fromDrive` folder, which are copies from B's Google Drive.
+`GEE_scripts` contains two scripts: one is the `era5_download`, which uses GEE to acquire the ERA5 modeled data for Yojoa and `guat_elsal_hon_getCenters`, which grabs the Chebyshev centers of all lakes greater than 25 ha in Guatemala, El Salvador, and Honduras.
 
-# Setting up the data symlink
+# Setting up the `data` folder symlink
 
-Link the applicable OneDrive folder to the `data` folder in the primary directory - your link will be different than the one in the folder. To link your data folder (basically a fancy shortcut), follow the instructions below where the code blocks are executed in the terminal. 
+Link the applicable OneDrive folder to the `data` folder in the primary directory - your link will be different than the one in the folder. To link your data folder (basically a fancy shortcut), follow the instructions below where the code blocks are executed in the terminal.
```
Comment (kathryn-willi) [inline README.md:27]: Link the applicable OneDrive folder to the `data` folder in the primary directory - your link will be different than the one in the folder (i.e., consider the data folder in this repository a placeholder). To link your data folder (basically a fancy shortcut), follow the instructions below where the code blocks are executed in the terminal.

File: programs/README.md [hunk 2]
```diff
@@ -1,30 +1,34 @@
-Script 1_RS_collate_harmonize.Rmd collates and harmonizes the output from GEE, 2_LSC2_secchi_matchup.Rmd creates the matchup files, X_LS4-7_LS8-9_handoff.Rmd is a quick model to aide in more transferable data between the two atmospheric correction methods.
+# yojoa-rs-secchi/programs
 
-Script 1_RS_collate_harmonize.Rmd assumes that you have run the Landsat stack files in the 'landsat_c2' folder within this repository or have access to the Drive folder with the output created in those scripts. You will not be able to run this code if you have not completed the LS stack run or if the output has not been shared with you.
+This directory contains the scripts that collate, harmonize, and generally munge the RS, Secchi, and met data into a common file. The programs are numbered in the order that they were originally run.
```
Comment (kathryn-willi) [inline programs/README.md:3]: meteorological (met)? my mind kept seeing this and automatically thinking it was meta just mispelled

File: programs/README.md [hunk 3]
```diff
@@ -1,30 +1,34 @@
...
+-   The files sameDay_LS-Secchi_matchups.csv, oneDay..., twoDay..., threeDay..., fiveDay... found in the data folder were created in the script 2_LSC2_secchi_matchup.Rmd 
 
-* `twoDay_LS-Secchi_matchups.csv`
 
-* `threeDay_LS-Secchi_matchups.csv`
 
-* `fiveDay_LS-Secchi_matchups.csv`
 
-* `Yojoa_Landsat_C2_SRST_collated_v2023-03-03.csv` : all data from the Landsat C2 scripts, no filters applied
+# Data set column definitions, units, sources
```
Comment (kathryn-willi) [inline programs/README.md:30]: very handy table 💯

**Review by kathryn-willi — APPROVED** [review]
One additional suggestion: it seemed unclear to me which subfolder's code needed to be run when. Maybe add numeric prefixes to the folders based on when to run each (like what you did within the `programs` README)? Or, if they aren't dependent upon each other, mention this in the primary README?

File: programs/README.md (hunk 2 above)
Comment (steeleb) [reply programs/README.md:3]: HA! cool - I'll try to catch those and spell them out!

File: programs/README.md (hunk 3 above)
Comment (steeleb) [reply programs/README.md:30]: Superb. It's was a dumb thing that we'll only ever have to do once, thank goodness!

**Comment by steeleb** [comment]
That's a great point, @kathryn-willi! I'll add a little text to the primary readme to help with this.

**Pull request merged**